src/oidcendpoint/endpoint.py

```diff
@@ -312,9 +312,6 @@ def do_post_parse_request(self, request, client_id="", **kwargs):
             request = meth(
                 request, client_id, endpoint_context=self.endpoint_context, **kwargs
             )
-            if isinstance(request, AuthorizationErrorResponse):
-                LOGGER.error(request.to_json())
-                raise UnAuthorizedClient(request.to_dict())
```

Comment:
Maybe should we return if we encounter error instead of invoking the next method?